Bottom: ca2b7a951a34d4aa27e8fd129f9ba1e705502dce
Top:    2d7e8769887cc91b75228e0012eabefe0e6fa280
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-09 05:13:25 +0000

Refresh of instrument-polygon-slowness

---

diff --git a/src/global.h b/src/global.h
index 1fb3420..157e186 100644
--- a/src/global.h
+++ b/src/global.h
@@ -261,12 +261,6 @@ typedef struct			/* holds information about arcs */
     Delta;
 } ArcType, *ArcTypePtr;
 
-typedef struct
-{
-  struct rtree_node *root;
-  int size;			/* number of entries in tree */
-} rtree_t;
-
 typedef struct			/* holds information about one layer */
 {
   char *Name;			/* layer name */
